app.go

```diff
@@ -101,6 +102,12 @@ func (a *APIApp) RestWrappers() {
 	a.wrappers = []Middleware{}
 }
 
+func (a *APIApp) AddCORS(opts cors.Options) *APIApp {
```

Comment:
sorry, to not put this in the earlier review.

This is awkward, because in order to use coors, you'll need to vendor the cors package anyway, which minimizes the benefit of this. I think it maybe makes sense to omit this. 

Also, that could push people to add cors middleware to only specific routes rather than the top level of the app (which might be good?)

Reply:
True, but I think the usefulness of this is if you want to enable this for all routes (which I think we will want to do for evergreen). This is mostly unnecessary if enabling for specific routes which I feel is fine

Reply:
Can you remove the interface dependency on the coors package?